01/mh70cz/wordvalue.py

```diff
@@ -0,0 +1,39 @@
+from data import DICTIONARY, LETTER_SCORES
+
+def load_words():
+    """Load dictionary into a list and return list"""
+    dl = []
+    with open(DICTIONARY) as f:
+        for line in f:
+            dl.append(line.strip())
+    return dl
+
+def calc_word_value(word):
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+    score = 0
+    for l in word:
+        l = l.upper()
+        score += LETTER_SCORES.get(l, 0)
+    return score
+
+
+def max_word_value(words=None):
+    """Calculate the word with the max value, can receive a list
+    of words as arg, if none provided uses default DICTIONARY"""
+    max_val = 0
+    max_val_word = None
+    if words is None:
```

Comment:
Nice use of `None` for argument checking. I also like this longer format solution as alternative to the shorter, but more cryptic `max` solution.